packages/nc-gui/composables/useProject.ts

```diff
@@ -35,7 +35,7 @@ export const useProject = createSharedComposable(() => {
 
   const project = ref<ProjectType>({})
 
-  const bases = computed<BaseType[]>(() => project.value?.bases || [])
+  const bases = computed<BaseType[]>(() => project.value?.bases?.filter((base: BaseType) => base.enabled) || [])
```

Comment:
By adding this change, it will break the share form views. You can reproduce the issue by creating a form view and then share it. The shared form view will be not shown and you can notice the errors from console.

The reason is that we haven't passed `enabled` in `useSharedFormViewStore`, hence `bases` would be empty (with your new change) causing undefined `sqlUis`.

https://github.com/nocodb/nocodb/blob/ea24fa91211f2bb1c6512504f5289b93b51833ee/packages/nc-gui/composables/useSharedFormViewStore.ts#L90-L98

Reply:
I figured that by requiring at least the main database to be enabled (if no others exist/ enabled) that wouldn't happen.